build_tools/kickstarts/rhel-7-cdk-vagrant.ks

```diff
@@ -58,6 +84,9 @@ openshift2nulecule
 # Workaround for BZ1336857
 restorecon -v /usr/bin/docker*
 
+LANG="en_US"
+echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
```

Comment:
I do not understand how this works. @praveenkumar @dustymabe do you guys have some pointers for me?